source/_integrations/device_tracker.markdown

```diff
@@ -97,7 +99,7 @@ In the example above, `devicename` refers to the detected name of the device.  F
 
 ## Device states
 
-The state of your tracked device will be `'home'` if it is in the [home zone](/integrations/zone#home-zone), detected by your network or Bluetooth based presence detection. If you're using a presence detection method that includes coordinates then when it's in a zone the state will be the name of the zone (case sensitive). When a device isn't at home and isn't in any zone, the state will be `'not_home'`.
+The state of your tracked device will be `'home'` if it is in the [home zone](/integrations/zone#home-zone), detected by your network or Bluetooth based presence detection. If you configure a `zone` for a network or Bluetooth based scanner, this zone is used instead of `'home'`, if the device is seen on the network. If you're using a presence detection method that includes coordinates then when it's in a zone the state will be the name of the zone (case sensitive). When a device isn't at home and isn't in any zone, the state will be `'not_home'`.
```

Comment:
For readability, I'd suggest switching the order around a bit here. Something along the lines of:
- If method includes coordinates, state will be the name of the zone
- If method doesn't include coordinates, state will be the name of the zone the scanner belongs to
- If not in any zone, state will be 'not_home'